### PR TITLE
Increased maximum image size to be uploaded as profile picture

### LINKED ISSUE #5041: [Bug] Max file size for profile pictures is too low
1. File size isn't mentioned before the user tries to upload a picture
2. File size of 244kb is too small because users will probably use a hi-res photo of themselves

### PR DESCRIPTION
**Pull request by nayan-rafiq**
## Description
We have a limit on the image size that can be uploaded as profile picture. The maximum allowed size is 244KB. This PR increases the allowed image size to 3MB in frontend and 1MB in backend.  

Fixes #5041 

## Type of change
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
- Unit tests
- Tested on local machine

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

File: app/client/src/components/ads/DisplayImageUpload.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default function DisplayImageUpload({
       allowMultipleUploads: false,
       restrictions: {
         maxNumberOfFiles: 1,
-        maxFileSize: 250000,
+        maxFileSize: 3145728, // 3 MB
```
Comment (pranavkanade) [inline app/client/src/components/ads/DisplayImageUpload.tsx:85]: @nayan-rafiq as suggested in https://github.com/appsmithorg/appsmith/issues/5041 can we show the file size limit warning?

**Review by pranavkanade — CHANGES_REQUESTED** [review]

File: app/client/src/components/ads/DisplayImageUpload.tsx (hunk 1 above)
Comment (mohanarpit) [reply app/client/src/components/ads/DisplayImageUpload.tsx:85]: Why have different values for image size in the client & server?

Comment (nayan-rafiq) [reply app/client/src/components/ads/DisplayImageUpload.tsx:85]: @mohanarpit We've an option to crop the selected image before uploading and we're also reducing the image quality before uploading. So a larger limit on front end will allow users to select larger images than the actual file that'll be uploaded.

Comment (mohanarpit) [reply app/client/src/components/ads/DisplayImageUpload.tsx:85]: But if the user doesn't crop the image, what will the image size end up being post compression? If we can predict that, it's OK to have separate limits. If we can't, then we'll run into issues where the user uploads a file within the client limits but the server rejects the HTTP request.

Comment (nayan-rafiq) [reply app/client/src/components/ads/DisplayImageUpload.tsx:85]: @mohanarpit Yeah, I've considered this. Uploaded image is 30% of the actual selected image. So if a 3MB file is selected, maximum uploaded file will be less than 1 MB. We've set 1MB limit on backend so it'll not be a problem.

**Comment by nayan-rafiq** [comment]
@pranavkanade I've added a message to show maximum allowed file size. It look like this one: https://uppy.io/examples/dashboard/
Would you please have a look on this?

**Review by pranavkanade — APPROVED** [review]
FE changes LGTM!

**Pull request merged**